model/command_response.go

```diff
@@ -5,10 +5,14 @@ package model
 
 import (
 	"encoding/json"
+	"errors"
+	"fmt"
 	"io"
 	"io/ioutil"
+	"net/url"
 	"strings"
 
+	"github.com/hashicorp/go-multierror"
```

Comment:
I don't have a strong option on this, but I'm wondering if `multierror` is really needed. Other `IsValid` method in this code base don't use it and consistency is important.

Reply:
For validation messages I've always like multierror because it gives the user all errors upfront. Otherwise, they'd have to continue making requests in order to see all of the errors. I can change this though, your call.